### ISSUE TITLE
Firefox Developer Edition 51.0a2 no video

### ISSUE THREAD

**Issue by thomaskieslich** [Selz/plyr#392]
In Firefox Developer Edition 51.0a2 no Video is shown only controls are visible.
It works with video.js or MEjs. 
I know it is a Developer Edition, but if nothing happens it will be a Problem with the next Firefox Version.

**Comment by SamPotts** [comment]
Will take a look. Thanks for raising this.

**Comment by tdaines** [comment]
I believe this is caused by the `webkit-mask-image` in `.plyr__video-wrapper`: https://github.com/Selz/plyr/blob/master/src/less/plyr.less#L178

Removing that from the style cause the video to appear in Firefox Developer Edition.

**Comment by SamPotts** [comment]
Thanks for this - about to push a fix for this

**Comment by SamPotts** [comment]
Fixed this by using `z-index: 0` instead. The mask was basically to ensure that border-radius was respected in WebKit. Weird that Mozilla was using their prefixed CSS but not unheard of. The issue now is that the `z-index: 0` breaks the YouTube embed and it's just black. I've raised a bug with Chrome for this:
https://bugs.chromium.org/p/chromium/issues/detail?id=658573
Demo: https://jsfiddle.net/Lnjqrt36/
